Makefile

```diff
@@ -87,33 +87,24 @@ doc-rust: setup-git ## generate API docs for Rust code
 
 # Style checking
 
-style: style-rust style-python ## check code style
```

Comment:
style and lint for python's moved into pre-commit

i decided to leave rust alone for now to keep things simple (will need some work as tools like rustfmt and clippy-driver are needed to accept individual file arguments)